src/applications/check-in/sass/check-in.scss

```diff
@@ -1,5 +1,6 @@
 @import "~@department-of-veterans-affairs/formation/sass/shared-variables";
 @import "~@department-of-veterans-affairs/css-library/dist/stylesheets/modules/m-action-link";
+@import "~@department-of-veterans-affairs/css-library/dist/stylesheets/utilities.css";
```

Comment:
Can this update be made here in the minimal stylesheet instead? 

https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/site-wide/sass/minimal.scss#L34-L35

Reply:
Done

Reply:
I added a GIF... it seems to work

Reply:
So moving the import from this `check-in.scss` file into the `minimal.scss` is moving the utilities _after_ the `_flexbox.scss` imports causing the flexbox definitions to override the utilities - [see screenshots in this comment](https://github.com/department-of-veterans-affairs/vets-website/pull/31901#issuecomment-2348829056).

Moving the import back into `check-in.scss` would fix the problem for this PR, but not globally. That is until the flexbox import is removed from `formation/core.scss`. So we need to find another solution.